### DISCUSSION TITLE
Solid-Solid heterostructure (Si/Dia) simulation issue at the interface of two materials

### DISCUSSION THREAD

**Discussion by kzadnan** [deepmodeling/deepmd-kit#3821] [Q&A]
Hello. I trained a deepmd potential for Si/Diamond interface. I also trained previously the potential on a MTP potential (it runs well with that potential) .But in deepmd, I am facing simulation error at the interface of two materials (mingling of atoms and liquid like beahvior). Could anyone help me on this? I have attached the test results and as well as the simulation error in lammps.
![Figure 2024-05-23 083008](https://github.com/deepmodeling/deepmd-kit/assets/150271459/c36b95f6-29d0-467b-92bd-7fecc3301496)
![Figure 2024-05-23 083038](https://github.com/deepmodeling/deepmd-kit/assets/150271459/209cc7be-1eba-4dfe-b3fa-9457952f92c6)
![Figure 2024-05-23 083044](https://github.com/deepmodeling/deepmd-kit/assets/150271459/f6b5039b-1c52-44bf-b1a0-182a36e9c218)
[input.json](https://github.com/deepmodeling/deepmd-kit/files/15443880/input.json)
[input_v2_compat.json](https://github.com/deepmodeling/deepmd-kit/files/15443881/input_v2_compat.json)
![INITIAL_STRUCTURE](https://github.com/deepmodeling/deepmd-kit/assets/150271459/4d9ca181-f7a3-4a2e-beca-5c45a876bf39)
![FINAL_STRUCTURE](https://github.com/deepmodeling/deepmd-kit/assets/150271459/9b47474c-55df-42cf-844b-5c452345c932)

**Comment by poyaii** [comment]
Hi . It seems like you have considered a smaller cut-off radius. Usually, in interface structures, a larger cut-off radius is considered. Have you performed the DFT-D3 calculations? Also, the structure used in AIMD is very important. Could you share all the structures (POSCARS ,Cifs ....) you used in the AIMD calculations so we can examine them further?

**Reply by kzadnan** [reply]
[INCAR.txt](https://github.com/deepmodeling/deepmd-kit/files/15448924/INCAR.txt)
[out.txt](https://github.com/deepmodeling/deepmd-kit/files/15448925/out.txt)
[POSCAR.txt](https://github.com/deepmodeling/deepmd-kit/files/15448927/POSCAR.txt)
 These are the aimd inputfile, output of results,  structure of the interface.

**Reply by kzadnan** [reply]
I am not sure about DFT-D3 calculations. I have tried larger cutoff but it did not work well.
